Mathlib/Analysis/BoxIntegral/Partition/Filter.lean

```diff
@@ -173,8 +173,7 @@ noncomputable section
 
 namespace BoxIntegral
 
-variable {ι : Type*} [Fintype ι] {I J : Box ι} {c c₁ c₂ : ℝ≥0} {r r₁ r₂ : (ι → ℝ) → Ioi (0 : ℝ)}
-  {π π₁ π₂ : TaggedPrepartition I}
+variable {ι : Type*} [Fintype ι] {I J : Box ι} {c c₁ c₂ : ℝ≥0}
```

Comment:
I don't understand why this change is needed: these are variables that can only be imported if they're mentioned in the type, right?

Reply:
Yes, it's not strictly necessary. I found it less confusing having these closer to the point of us.